src/devtools/client/debugger/src/actions/pause/paused.js

```diff
@@ -18,9 +19,10 @@ import { setFramePositions } from "./setFramePositions";
  * @memberof actions/pause
  * @static
  */
-export function paused({ executionPoint }) {
+export function paused({ executionPoint, time }) {
   return async function ({ dispatch, getState }) {
     dispatch({ type: "PAUSED", executionPoint });
+    dispatch(actions.setPauseExecutionPoint(executionPoint, time));
```

Comment:
could we go one step further and get rid of `setPauseExecutionPoint`?

 ```
dispatch({ type: "PAUSED", executionPoint, time });
```

and in `reducers/messages` handle the `PAUSED` action. Having fewer actions is easier to understand